python/ispy_GEMs_cfg.py

```diff
@@ -0,0 +1,86 @@
+import os
+
+# Stuff for singularity on lxplus
+outPath = os.getenv('/afs/cern.ch/user/s/seungjun/public/ISPY/CMSSW_11_2_3/src')
+
+if not outPath:
+  outPath = ''
+else:
+  outPath += '/'
+
+import FWCore.ParameterSet.Config as cms
+from Configuration.StandardSequences.Eras import eras
+process = cms.Process("ISPY", eras.Run3)
+process.load("Configuration.StandardSequences.GeometryDB_cff")
+process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
+##process.GlobalTag.globaltag = '112X_mcRun3_2021_design_v14'
+##process.GlobalTag.globaltag = '112X_dataRun3_Prompt_v5' 
+##process = cms.Process("ISPY")
+
+process.load("Configuration.StandardSequences.MagneticField_cff")
+
+#from Configuration.AlCa.GlobalTag_condDBv2 import GlobalTag
+from Configuration.AlCa.GlobalTag import GlobalTag                                ##
+process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:phase1_2021_realistic', '')## 
+
+import FWCore.Utilities.FileUtils as FileUtils
+
+process.source = cms.Source(
+    'PoolSource',
+    fileNames = cms.untracked.vstring(
+    #'root://cmsxrootd.fnal.gov//store/data/Run2018D/DoubleMuon/AOD/PromptReco-v2/000/324/998/00000/AF519538-7FE5-4A4B-BD66-6FE4900CB5C6.root'
+    # '/store/data/Commissioning2021/Cosmics/RAW-RECO/CosmicTP-PromptReco-v1/000/341/169/00000/095946f3-e67a-4b4f-a098-3432372b6d0e.root'
+   #  '/store/group/alca_muonalign/run3MC/step2_501.root'
+     'root://cms-xrdr.private.lo:2094//xrd/store/user/seungjun/ISPY/display_out_full.root'
```

Comment:
Perhaps just keep the two files here that you used to produce the ig files and rm the other ones?

Reply:
i keep one and remove the others